code/datums/climate.dm

```diff
@@ -0,0 +1,209 @@
+/********************************************
+*                IN THIS FILE               *
+*          Climate Datum, Weather Datum     *
+*                                           *
+********************************************/
+
+#define PREDICTION_MINIMUM 4
+#define PREDICTION_MAXIMUM 10
```

Comment:
Document what these mean so when you or someone else come back to fix it ten years from now, you'll know what the fuck these do.